Tasks/AzureRmWebAppDeployment/operations/AzureAppServiceUtility.ts

```diff
@@ -63,13 +63,21 @@ export class AzureAppServiceUtility {
     }
 
     public async pingApplication(): Promise<void> {
-        try {
-            var applicationUrl: string = await this.getApplicationURL();
+        var applicationUrl: string = await this.getApplicationURL();
```

Comment:
I think you are changing the behavior by not ignoring the errors in getApplication

Reply:
reverted back to the earlier behavior.